cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/13-school-results.html

```diff
@@ -56,7 +56,7 @@ <h3 class="u-border-bottom"><span data-school-item="school">Example University</
             'This school is under Heightened Cash Monitoring by the U.S. Department of Education. <a href="https://studentaid.gov/data-center/school/htm" target="_blank" class="external-link" rel="noreferrer noopener">Find out more.</a>' | safe
         ) }}
 
-        <div class="undergrad-content">
+        <div class="undergrad-content" data-state-based-visibility="none">
```

Comment:
This content is still showing for me -- looks like the undergrad rule is overriding the visibility one

Reply:
I just realized I was testing this in exactly the wrong way, so thanks for catching that! I'll push a fix.